### DISCUSSION TITLE
嵌入数学公式后文本无法加粗或斜体

### DISCUSSION THREAD

**Discussion by PearlLHF** [wangchucheng/hugo-eureka#98] [Q&A] [answered]
您好！我在编辑文档的时候遇到了一个问题：

文档中插入inline equation之后，有些公式后面的文本无法加粗或斜体，换行之后才能正常渲染，如图所示，
md源码：
![image](https://user-images.githubusercontent.com/43177195/116985150-ad0e5380-acfe-11eb-8419-7dc985296279.png)
渲染效果：
![image](https://user-images.githubusercontent.com/43177195/116985179-b4cdf800-acfe-11eb-8924-e4c5cf8ce967.png)

想请教一下如何解决这类问题？

谢谢！

**Comment by angelajt** [comment, marked as answer]
Try putting `\` before the `_` in your equation. :relaxed: 

`$h_{t-1}$` → `$h\_{t-1}$`

![Screenshot from 2021-05-08 11-53-24](https://user-images.githubusercontent.com/10780322/117550303-6d908180-aff4-11eb-9a79-d8f723265a40.png)
![Screenshot from 2021-05-08 11-53-33](https://user-images.githubusercontent.com/10780322/117550305-708b7200-aff4-11eb-8179-47e770288035.png)

**Reply by PearlLHF** [reply]
It works! Thank you🙏 ☺
